tests/store/artifact/test_databricks_artifact_repo.py

```diff
@@ -0,0 +1,137 @@
+# -*- coding: utf-8 -*-
```

Comment:
Awesome tests! One more small ask: Can we add tests for the `_extract_run_id` method to ensure that the expected run ID is extracted properly for a variety of DBFS URIs (perhaps the same URIs used in the test cases you added to `test_uri.py`)?

Reply:
Done.